src/app/item-popup/PlugTooltip.tsx

```diff
@@ -49,21 +51,64 @@ export default function PlugTooltip({
       )
     : [];
 
+  const stats: { [statHash: string]: number } = {};
+
+  for (const statHash of visibleStats) {
+    const value = plug.stats?.[parseInt(statHash, 10)];
+    if (typeof value === 'number') {
+      stats[statHash] = value;
+    }
+  }
+
+  return (
```

Comment:
Would be good to explain what the relationship between `PlugTooltipContent` and `PlugTooltip` is, now.

Reply:
Can do